index.d.ts

```diff
@@ -0,0 +1,42 @@
+declare class Collapsible extends React.Component<CollapsibleProps> {}
+declare namespace Collapsible {}
+
+interface CollapsibleProps extends React.HTMLProps<Collapsible> {
+  transitionTime?: number
+  transitionCloseTime?: number | null
+  triggerTagName?: string
+  easing?: string
+  open?: boolean
+  classParentString?: string
+  openedClassName?: string
+  triggerStyle?: null | Object
+  triggerClassName?: string
+  triggerOpenedClassName?: string
+  contentOuterClassName?: string
+  contentInnerClassName?: string
+  accordionPosition?: string | number
+  handleTriggerClick?: (accordionPosition?: string | number) => void
+  onOpen?: () => void
+  onClose?: () => void
+  onOpening?: () => void
+  onClosing?: () => void
+  trigger: string | React.ReactElement<any>
+  triggerWhenOpen?: string | React.ReactElement<any>
+  triggerDisabled?: false
```

Comment:
could this be a mistake? I don't think that `?: false` is the correct type. As of now, you cannot set `triggerDisabled` to `true`. I would understand `?: true`, as this would be `true | undefined`, but I think `?: boolean` should suffice ...

Reply:
This is definitely a mistake, why would it always need to be false?